rust/cubesql/cubesql/src/sql/postgres/pg_auth_service.rs

```diff
@@ -87,15 +87,14 @@ impl PostgresAuthService for PostgresAuthServiceDefaultImpl {
             )
             .await;
 
-        let auth_fail = || {
-            AuthenticationStatus::Failed(format!(
-                "password authentication failed for user \"{}\"",
-                user
-            ))
-        };
-
-        let Ok(authenticate_response) = authenticate_response else {
-            return auth_fail();
+        let authenticate_response = match authenticate_response {
+            Ok(r) => r,
+            Err(err) => {
+                return AuthenticationStatus::Failed(format!(
+                    "password authentication failed for user \"{}\". \nError: {}",
```

Comment:
It's not safe to expose all error messages to end user. For example:

user can define `check_sql_auth` with HTTP client and it can throw something secret in exception's message. Like URL of JWT keys or database URL, which contains all parameters.

I belive, we should change message, that error is happening, please take a look at logs. And print it to logs as error.

cC @igorlukanin